examples/message_request_response_handling.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+import json
+import sys
+import threading
+
+import paho.mqtt.client as mqtt
+from ditto.client import Client
+from ditto.model.namespaced_id import NamespacedID
+from ditto.protocol.envelope import Envelope
+from ditto.protocol.things.messages import Message
+from ditto.model.feature import Feature
+from ditto.model.definition_id import DefinitionID
+
+
+thing_id = NamespacedID().from_string("test.ns:test-name")
+
+feature_id = "MyFeatureID"
+property_id = "myProperty"
+definition_id = DefinitionID().from_string("my.model.namespace:FeatureModel:1.0.0")
+my_feature = Feature().with_definition(definition_id).with_property(property_id, "myValue")
+req_topic = "command///req/" + str(thing_id) + "/"
+message_subject = "some-command"
+
+
+def send_inbox_message():
+    live_message = Message(thing_id).inbox(subject=message_subject).with_payload("some_payload")
```

Comment:
Should be possible to use the short notation for the .inbox(message_subject) call here.